### PR TITLE
Adds support for subdirectories when reading modules

### LINKED ISSUE #663: Mirage fails when models are in sub-folders
Using 0.2.0-beta.8
`ember g mirage-model admin/log`

ember.debug.js:32127Error: Assertion Failed: Subdirectories under admin are not supported
    at new Error (native)
    at Error.EmberError (http://localhost:4200/assets/vendor.js:26056:21)
    at assert (http://localhost:4200/assets/vendor.js:15815:13)
    at Object.assert (http://localhost:4200/assets/vendor.js:25820:34)
    at http://localhost:4200/assets/vendor.js:117769:25
    at Array.forEach (native)
    at exports.default (http://localhost:4200/assets/vendor.js:117761:8)
    at startMirage (http://localhost:4200/assets/client.js:415:66)
    at Object.initialize (http://localhost:4200/assets/client.js:406:9)
    at http://localhost:4200/assets/vendor.js:14823:23

### PR DESCRIPTION
**Pull request by jrhe**
Should fix #663 #540. Needs testing but wanted some guidance on what the test cases should be and where they should go to suit the project. All existing tests are green.

### CODE REVIEW

File: addon/db.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Db {
    * @public
    */
   loadData(data) {
+    console.log(data);
```
Comment (samselikoff) [inline addon/db.js]: could we remove this?

**Comment by samselikoff** [comment]
Nice! I would say look at the node-tests. A single test to check that a nested module was loaded would be a great start

**Comment by acorncom** [comment]
@jrhe We're working on tidying up old issues / PRs.  Do you think you'll be able to come back to this or should we close this PR for now?

**Comment by jrhe** [comment]
Can you close it for now? I don't think I'll have time any time soon.

**Comment by acorncom** [comment]
Will do 👍

**Pull request closed without merging**